guides/common/modules/proc_configuring-a-host-for-registration.adoc

```diff
@@ -1,44 +1,38 @@
 [id="Configuring_a_Host_for_Registration_{context}"]
 = Configuring a Host for Registration
 
-{RHEL} virtual machines register to Customer Portal Subscription Management by default.
-You must update each virtual machine configuration so that they receive updates from the correct {Project} Server or {SmartProxyServer}.
+Configure your host for registration to {ProjectServer} or {SmartProxyServer}.
 
 .Prerequisites
-* Hosts must be using the following {EL} version:
-** 6.4 or later
-** 7.0 or later
-* All architectures of {EL} are supported (i386, x86_64, s390x, ppc_64).
-* Ensure that a time synchronization tool is enabled and runs on the {ProjectServer}s, any {SmartProxyServers}, and the hosts.
-** For {EL} 6:
-+
-----
-# chkconfig ntpd on; service ntpd start
-----
-** For {EL} 7:
+* The host must be using a supported OS version.
+For more information, see xref:supported-clients-in-registration_{context}[].
+
+.Procedure
+. Ensure that a time-synchronization tool is enabled and running on the host.
+ifdef::client-content-dnf[]
+** For {EL} 7 and newer:
 +
 ----
 # systemctl enable --now chronyd
 ----
-* Ensure that the daemon `rhsmcertd` is enabled and running on the hosts.
 ** For {EL} 6:
 +
 ----
-# chkconfig rhsmcertd on; service rhsmcertd start
+# chkconfig ntpd on
+# service ntpd start
 ----
-** For {EL} 7:
+endif::[]
+. Ensure that the `rhsmcertd` daemon is enabled and running on the host.
+ifdef::client-content-dnf[]
+** For {EL} 7 and newer:
 +
 ----
 # systemctl enable --now rhsmcertd
 ----
-
-.To Configure a Host for Registration:
-. Take note of the fully qualified domain name (FQDN) of the {ProjectServer} or {SmartProxyServer}.
-. On the host, connect to a terminal on the host as the root user
-. Install the consumer RPM from {ProjectServer} or {SmartProxyServer} to which the host is to be registered.
-The consumer RPM updates the content source location of the host and allows the host to download content from the content source specified in {ProjectName}.
+** For {EL} 6:
 +
-[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# rpm -Uvh http://{foreman-example-com}/pub/katello-ca-consumer-latest.noarch.rpm
+# chkconfig rhsmcertd on
+# service rhsmcertd start
```

Comment:
I guess the service should be also enabled, so it starts automatically after the reboot.

Reply:
Like this?